### PR TITLE
Fix linting issues

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

File: manager/resource.go [hunk 1]
```diff
@@ -15,11 +15,11 @@
 package manager
 
 import (
-v1 "k8s.io/api/core/v1"
-"k8s.io/apimachinery/pkg/labels"
-corev1listers "k8s.io/client-go/listers/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	corev1listers "k8s.io/client-go/listers/core/v1"
```
Comment (sargun) [inline manager/resource.go]: Why is there a newline here?

File: provider/mock/mock.go [hunk 2]
```diff
@@ -354,8 +354,8 @@ func (p *MockV0Provider) GetPods(ctx context.Context) ([]*v1.Pod, error) {
 	return pods, nil
 }
 
-func (p *MockV0Provider) ConfigureNode(ctx context.Context, n *v1.Node) {
-	ctx, span := trace.StartSpan(ctx, "mock.ConfigureNode")
+func (p *ProviderV0) ConfigureNode(ctx context.Context, n *v1.Node) {
+	_, span := trace.StartSpan(ctx, "mock.ConfigureNode")
```
Comment (sargun) [inline provider/mock/mock.go]: This is somewhat of an aside, but why do all of the other spans just have the function name, whereas this is mock.ConfigureNode?

Comment (cpuguy83) [reply provider/mock/mock.go]: 🤷‍♂

File: manager/resource.go (hunk 1 above)
Comment (cpuguy83) [reply manager/resource.go]: Probably goimports did something it shouldn't have done. Let me fix that.

File: provider/mock/mock.go (hunk 2 above)
Comment (cpuguy83) [reply provider/mock/mock.go]: Updated to make this like everything else.

File: manager/resource.go (hunk 1 above)
Comment (cpuguy83) [reply manager/resource.go]: Fixed

File: provider/mock/mock.go [hunk 3]
```diff
@@ -257,15 +257,15 @@ func (p *MockV0Provider) DeletePod(ctx context.Context, pod *v1.Pod) (err error)
...
+	pod.Status.Reason = "ProviderPodDeleted"
 
 	for idx := range pod.Status.ContainerStatuses {
 		pod.Status.ContainerStatuses[idx].Ready = false
 		pod.Status.ContainerStatuses[idx].State = v1.ContainerState{
 			Terminated: &v1.ContainerStateTerminated{
 				Message:    "Mock provider terminated container upon deletion",
 				FinishedAt: now,
-				Reason:     "MockProviderPodContainerDeleted",
+				Reason:     "ProviderPodContainerDeleted",
```
Comment (sargun) [inline provider/mock/mock.go]: This provider is replicated between the node-cli repo, and the VK repo, why?

Comment (cpuguy83) [reply provider/mock/mock.go]: I think the intention was to have mock live here. It's been so long at this point, though.

**Pull request merged**